elastic/kibana
# [Discover] Fix navigating back when changing index pattern

## Summary

This PR restores the functionality of Discover to navigate back and forth selected index patterns using the browser back and forward button.

### How to test

1. Go to Discover
2. Change index pattern
3. Click browser back button -> now the previous index pattern should be selected

Fixes #79872

### Checklist
